tests/es6-timed-map.spec.ts

```diff
@@ -1,7 +1,7 @@
 import Es6TimedMap from '../src/es6-timed-map';
 
 describe('Es6TimedMap', () => {
-  let timedMap: Es6TimedMap<string, string>;
+  let timedMap: Es6TimedMap<any, any>;
```

Comment:
If your asking about this, I set the 2 definitions for the tests as strings for simplicity sake, but `any` isn't supported due to the linter, so keeping it as a string should be fine **or** we can turn off the `any` check so we can do whatever during these tests

Reply:
If we change it to string, not sure how we would test for Symbol/Object keys.
> A Map's keys can be any value (including functions, objects, or any primitive).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
we could just create a new `timedMap` in those tests with a different type-def, I had one main one just for simplicity stake for **most** tests.